src/plugins/intel_gna/src/gna_transformations_pipeline.cpp

```diff
@@ -180,6 +267,39 @@ void TransformationsPipeline::apply(const std::shared_ptr<ov::Model>& model,
     manager.run_passes(model);
 
     is_ngraph_passes_used = true;
+}  // namespace intel_gna
+
+void TransformationsPipeline::register_all_passes(const PassesInstances& passes, ov::pass::Manager& manager) {
+    for (const auto& pass : passes) {
```

Comment:
what does const give us in this context? it should have been deduced here right?